examples/src/bin/twai.rs

```diff
@@ -13,6 +13,9 @@
 //! ESP1/GPIO0 --- ESP1/GPIO2 --- ESP2/GPIO0 --- ESP2/GPIO2 --- 4.8kOhm --- ESP1/5V
 //!
 //! `IS_FIRST_SENDER` below must be set to false on one of the ESP's
+//!
+//! In case you want to use `self-testing``, get rid of everything related to the aforementioned `IS_FIRST_SENDER`
```

Comment:
this was a bit misleading to me since the unmodified example here is already doing self-reception?

Not sure if we should have it like that b.t.w. - shouldn't we just use self-reception for HIL testing? Is it ever useful for a real application?

Reply:
Oops, I forgot to return the example to its original functionality after tests. Fixed!

Yes, the main reason why we want `self-testing` is that it will greatly simplify HIL tests. There is no practical application of this, I think. Do you think I should remove this part of a doc comment?